### DISCUSSION TITLE
Place Order Missing in PlatformUI

### DISCUSSION THREAD

**Discussion by bloodnovski** [inventree/InvenTree#8379] [Q&A]
![1](https://github.com/user-attachments/assets/df56b499-bf2d-43e3-843d-b3e555f84cec)
![2](https://github.com/user-attachments/assets/b6511cde-9a85-43ed-a031-33386f904f81)

**Comment by SchrodingersGat** [comment]
@bloodnovski the new user interface is under active development, so please make sure that you check the demo server first, which is running the latest code:

e.g. from https://demo.inventree.org/platform/purchasing/purchase-order/15/detail

![image](https://github.com/user-attachments/assets/c690cc86-8fec-457f-b5f4-971c70ca3b33)
